Source/Fuse.Reactive.JavaScript/Tests/UX/OptionalExplicit.ux

```diff
@@ -0,0 +1,21 @@
+<Panel ux:Class="UX.OptionalExplicit">
+	<Router ux:Name="router" />
+	<JavaScript>
+		exports.foo = 123
+		// should not be reached below, as there is a bundled file that matches
+		// this path
+		exports.OptionalExplicit = 777 
```

Comment:
It is visible below because of dependency injection:  `<JavaScript dep:bar="{foo}">`